### PR TITLE
Intro edits

### PR DESCRIPTION
**Pull request by mariaruth**

### CODE REVIEW

**Comment by mariaruth** [comment]
@bbdaniels i did quite a bit of reorganization to try to improve the logical flow of the sections and paragraphs, and overall make the intro tighter. please read and let me know what you think.

File: chapters/0-introduction.tex [hunk 1]
```diff
@@ -17,18 +16,30 @@
 This means that the many different people on a team need to have the same ideas
 about what is to be done, and when and where and by whom,
 so that they can collaborate effectively on a large, long-term research project.
-This book contains our ideas on how to perform the main tasks
-of a complex research project using original data.
+
+Despite the growing importance, little practical guidance is available for practitioners
```
Comment (bbdaniels) [inline chapters/0-introduction.tex:20]:
```suggestion
Despite the growing importance of managing data work, 
little practical guidance is available for practitioners
```

File: chapters/0-introduction.tex [hunk 2]
```diff
@@ -65,16 +76,28 @@ \section{Who is the team behind this book?}
...
-See the DIME Analytics Resources Appendix for an introduction to our materials.
-
-\textit{Development Research in Practice} is not sector-specific;
-it will not teach you econometrics,
-or how to design an impact evaluation.
-There are many excellent existing resources on those topics.
-Instead, this book will teach you how to think about all aspects of your research from a data perspective,
-how to structure research projects to ensure data quality,
-and how to institute transparent and reproducible workflows.
+The DIME Analytics Resources Appendix provides an introduction to public materials.
```
Comment (bbdaniels) [inline chapters/0-introduction.tex:79]:
```suggestion
The DIME Analytics Resource Directory appendix provides an introduction to public materials.
```

**Review by bbdaniels — CHANGES_REQUESTED** [review]
I think this looks great! Only two very minor edits.

**Comment by mariaruth** [comment]
@bbdaniels - i accepted both changes, if nothing else let's merge this

**Review by bbdaniels — APPROVED** [review]

**Pull request merged**